corehq/apps/hqwebapp/migrations/0009_truncate_authtoken_table.py

```diff
@@ -8,5 +8,10 @@ class Migration(migrations.Migration):
     ]
 
     operations = [
-        migrations.RunSQL("TRUNCATE TABLE authtoken_token", reverse_sql=migrations.RunSQL.noop)
+        migrations.RunSQL("""
+            IF EXISTS (SELECT 1 FROM INFORMATION_SCHEMA.TABLES WHERE TABLE_NAME = 'authtoken_token')
+            begin
+                TRUNCATE TABLE authtoken_token;
+            end
```

Comment:
we have a pattern for doing this already: https://github.com/dimagi/commcare-hq/blob/865028a1448c94cb5826f18b99aff36b43f72944/corehq/util/django_migrations.py#L30-L34